src/main/webapp/app/modules/games/single-game/single-game.tsx

```diff
@@ -0,0 +1,128 @@
+import '../games.scss';
+
+import {RouteComponentProps} from "react-router-dom";
+import {connect} from "react-redux";
+import {IRootState} from "app/shared/reducers";
+import {getSingleGame} from "app/modules/games/games.reducer";
+import React, {useEffect, useState} from "react";
+import {Col, Container, DropdownItem, Row} from 'reactstrap';
+import {cleanText, formatCurrency} from "app/shared/util/render-utils";
+import {CartQuantityBox} from "app/modules/games/single-game/cart-quantity-box";
+import {CategoriesMechanics} from "app/modules/games/single-game/categories-mechanics";
+import {Carousel} from "app/modules/games/single-game/carousel";
+
+export interface ISingleGameProps extends StateProps, DispatchProps, RouteComponentProps<{ id: string }> {
+}
+
+export const SingleGame = (props: ISingleGameProps) => {
+  const [formattedCurrency, setFormattedCurrency] = useState("");
```

Comment:
Is there a reason this needs to be stored in state? Can you just call the formatCurrency method on price from within the render method? As far as I can tell, price is never going to change from within a child component, so getting it from the props should be enough

Reply:
You're right - we don't need this so I took it out 👍